### PR TITLE
Port of MPB test 1b from linproginterface

### PR DESCRIPTION
**Pull request by IssamT**

### CODE REVIEW

File: test/contlinear.jl [hunk 1]
```diff
@@ -292,6 +292,67 @@ function contlineartest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64)
         end
     end
 
+    @testset "addvariable! (one by one) interface" begin
+        # Min -x
+        # s.t. x + y <= 1
+        # x, y >= 0
+
+        @test MOI.supportsproblem(solver, MOI.ScalarAffineFunction, [(MOI.ScalarAffineFunction{Float64},MOI.LessThan),(MOI.ScalarVariablewiseFunction,MOI.GreaterThan)])
```
Comment (mlubin) [inline test/contlinear.jl]: Missing type parameters on `MOI.ScalarAffineFunction` and the sets. (Yes, we're not yet consistent everywhere.)

File: test/contlinear.jl [hunk 2]
```diff
@@ -292,6 +292,67 @@ function contlineartest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64)
...
+        m = MOI.SolverInstance(solver)
+
+        x = MOI.addvariable!(m)
+        y = MOI.addvariable!(m)
+
+        @test MOI.getattribute(m, MOI.NumberOfVariables()) == 2
+
+        cf = MOI.ScalarAffineFunction([x, y], [1.0,1.0], 0.0)
+        c = MOI.addconstraint!(m, cf, MOI.LessThan(1.0))
+        @test MOI.getattribute(m, MOI.NumberOfConstraints{MOI.ScalarAffineFunction{Float64},MOI.LessThan}()) == 1
```
Comment (mlubin) [inline test/contlinear.jl]: Missing type parameter on `MOI.LessThan`

File: test/contlinear.jl [hunk 3]
```diff
@@ -292,6 +292,67 @@ function contlineartest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64)
...
+
+        vc1 = MOI.addconstraint!(m, MOI.ScalarVariablewiseFunction(x), MOI.GreaterThan(0.0))
+        vc2 = MOI.addconstraint!(m, MOI.ScalarVariablewiseFunction(y), MOI.GreaterThan(0.0))
+        @test MOI.getattribute(m, MOI.NumberOfConstraints{MOI.ScalarAffineFunction{Float64},MOI.GreaterThan}()) == 2
```
Comment (mlubin) [inline test/contlinear.jl]: Missing type parameter on `MOI.GreaterThan`

File: test/contlinear.jl [hunk 4]
```diff
@@ -292,6 +292,67 @@ function contlineartest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64)
...
+
+        objf = MOI.ScalarAffineFunction(v, [-1.0,0.0], 0.0)
```
Comment (joaquimg) [inline test/contlinear.jl]: `v` is not defined

File: test/contlinear.jl (hunk 1 above)
Comment (joaquimg) [inline test/contlinear.jl]: Wait, you mean we should not add `{Float64}` in `MOI.LessThan` in `MOI.supportsproblem`?

Comment (mlubin) [inline test/contlinear.jl]: We should add `{Float64}` for `MOI.supportsproblem`. It's not nice to call it with non-concrete types.

**Pull request merged**